eth_account/signers/local.py

```diff
@@ -158,3 +158,9 @@ def sign_typed_data(
                 full_message=full_message,
             ),
         )
+
+    def sign_authorization(self, authorization: Dict[str, Any]) -> SignedMessage:
```

Comment:
Is there a use case where we'd want to pass an authorization list here? For example either a new method something like `sign_authorization_list` or `sign_authorizations(self, authorizations: List[Dict[str, Any]]) ...`

Reply:
Authorizations can be signed by different accounts. I think that would abstract a fairly small step.

```py
authorization_list = [
    acct.sign_authorization(auth).as_rpc_object() for auth in (
        {"address": ..., "nonce": ..., "chainId": ...},
        {"address": ..., "nonce": ..., "chainId": ...},
    )
]
```

I lean towards keeping it focused on single auths tbh. Maybe we can see if there's demand first.

Reply:
That sounds like a good plan to me! 👍